ocs_ci/ocs/monitoring.py

```diff
@@ -132,3 +132,34 @@ def check_pvcdata_collected_on_prometheus(pvc_name):
         )
     logger.info(f"Created pvc {pvc_name} data {pvc_list} is collected on prometheus pod")
     return True
+
+
+def prometheus_health_check(name='monitoring', kind='ClusterOperator'):
+    """
+    Return true if the prometheus cluster is healthy
+
+    Args:
+        name (str) : Name of the resources
+        kind (str): Kind of the resource
+
+    Returns:
+         bool : True on prometheus health is ok, false otherwise
+
+    """
+    ocp_obj = OCP(kind=kind)
+    health_info = ocp_obj.get(resource_name=name)
+    health_conditions = health_info.get('status').get('conditions')
+
+    # Check prometheus is degraded
+    # If degraded, degraded value will be True, AVAILABLE is False
+    for i in health_conditions:
+        if {('type', 'Available'), ('status', 'True')}.issubset(set(i.items())):
+            available = True
+        if {('status', 'False'), ('type', 'Degraded')}.issubset(set(i.items())):
+            degraded = True
+
+    if available and degraded:
+        return True
```

Comment:
The return is inside the 'if' statement so if there will be more than 1 conditions, you might return after the first one